src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1414,6 +1413,11 @@ void PosixSignals::print_signal_handler(outputStream* st, int sig,
   sigaction(sig, NULL, &current_act);
 
   print_single_signal_handler(st, &current_act, buf, buflen);
+
+  sigset_t thread_sig_mask;
+  if (::pthread_sigmask(/* ignored */ SIG_BLOCK, NULL, &thread_sig_mask) == 0) {
```

Comment:
Nit: no need for `::` scoping (there is one other existing unnecessary use as well you could remove :) ).